### ISSUE TITLE
Audio issues

### ISSUE THREAD

**Issue by wallace4BR** [SkyCraft78/DiSky3#57]
https://user-images.githubusercontent.com/81590228/141689878-05bb0e4d-3f3d-4c96-9a45-fbd556f5dea9.mp4

Playing song with two different bots stills makes the bot play the track 2x faster
Code used

```code
discord command -play:
	bots: wBot
	trigger:
		search youtube for "Happier" and store the results in {_results2::*}
		set {_guild} to guild with id "888815146980503682" with bot "wBot"
		connect bot "wBot" to voice channel of event-member
		play {_results2::1} in {_guild} with bot "wBot"

discord command -play2:
	bots: wBotBeta
	trigger:
		search youtube for "On&On" and store the results in {_results::*}
		set {_guild2} to guild with id "888815146980503682" with bot "wBotBeta"
		connect bot "wBotBeta" to voice channel of event-member
		play {_results::1} in {_guild2} with bot "wBotBeta"
```
Related issue since #31

**Comment by SkyCraft78** [comment]
Hello, thanks for your issue :)
Not sure through so I prefer asking;
When the second bot connects, it's both bot that good speed up or only the connected one?
In addition, when the bot connects (the second one), isn't there any message in the console? (LavaDSP should message the filter loading if it was really used here)

**Comment by wallace4BR** [comment]
> Hello, thanks for your issue :) Not sure through so I prefer asking; When the second bot connects, it's both bot that good speed up or only the connected one? In addition, when the bot connects (the second one), isn't there any message in the console? (LavaDSP should message the filter loading if it was really used here)

When the second bot connects, this bot plays with the speed up too and also it starts playing the track from the first bot along with the position that its currently playing
And no, theres not any message in console, here is some debugs that i made that may help you
![image](https://user-images.githubusercontent.com/81590228/142066790-c6fc6e9e-1de1-41b5-abdb-5f13a4254dd4.png)
It seens like lavaplayer is not creating a new channel/player for the second bot and is using the primary one?

**Comment by wallace4BR** [comment]
Fixed in 3.7
